docs-markdown/markdownlint-custom-rules/common.js

```diff
@@ -0,0 +1,25 @@
+// @ts-check
+
+"use strict";
+
+// Triple colon
+module.exports.singleColon = /^:/gm;
+module.exports.tripleColonSyntax = /^:::/gm;
+module.exports.validTripleColon = /^:::\s+/gm;
```

Comment:
Does space at the beginning of the `:::` make it invalid?

Reply:
Hi @nschonni , currently the ::: should start at the beginning of a line with no whitespace.  Not sure if that may change going forward.  Thanks

Reply:
👍 should there be a test for the the triple colon with leading spaces as a failure then?

Reply:
Good call, I'll let the PM know for follow-up.  Thanks